packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [60.0.0]
+
+### Added
+
+- Add support for 'Sonic Mainnet' chainId in the list of SUPPORTED_CHAIN_IDS. ([#5711](https://github.com/MetaMask/core/pull/5711))
```

Comment:
@sahar-fehri do we need this release ? this change has already been released part of the previous release ? https://github.com/MetaMask/core/pull/5717/files#diff-ee47d03d53776b8dd530799a8047f5e32e36e35765620aeb50b294adc3339fabR32

Reply:
Actually i think v59 does not have the change 
https://github.com/MetaMask/core/pull/5711
Something weird happened on that PR where i enabled auto-merge; and i merged main (which resulted in a changelog issue) but the PR got merged even though pipeline was red.

Reply:
Okay I see can you in this case remove the changelog entry from previous version

Reply:
currenctly check-changelog step is not a blocker to merge the changes.

Reply:
Done here [019093f](https://github.com/MetaMask/core/pull/5717/commits/019093f075298a2636112adde95b21a66caa2caf)